ligo/src/registryDAO/types.mligo

```diff
@@ -16,10 +16,20 @@ type update_receiver_param =
   | Add_receivers of (address list)
   | Remove_receivers of (address list)
 
+// For reference only
+// ```
+// type proposal_metadata =
+//   [@layout:comb]
+//   { agora_post_id : nat
+//   ; transfers : transfer_type list
+//   }
+// ```
```

Comment:
:thinking: this is only for one type of proposal tho, a correct full representation it seems to me that it should look like:
```suggestion
// For reference only
// ```
// type proposal_metadata_transfers =
//   { agoraPostID : nat
//   ; transfers : transfer_type list
//   }
//
// type proposal_metadata_updates =
//   { agoraPostID : nat
//   ; updates : registry_diff
//   }
//
// type proposal_metadata_update_receivers =
//   { update_receivers : update_receiver_param
//   }
//
// type proposal_metadata =
//   | Normal_metadata of proposal_metadata_updates
//   | Update_receivers_metadata of update_receiver_param
//   | Transfers_metadata of proposal_metadata_transfers
//   | Configuration_metadata
// ```
```

Reply:
Ah, sorry, I noticed I hadn't actually understood what I had to do now that you made your suggestion. I updated it with your proposed changes.